src/App.tsx

```diff
@@ -10,8 +10,8 @@ function App() {
   const [editingIdValue, setEditingIdValue] = useState<string>('');
   const [inputValue, setInputValue] = useState<string>('');
 
-  const addTask = (task: string, newKey: string): void => {
-    setTaskList([...taskList, { value: task, id: newKey }]);
+  const addTask = (task: string, newKey: string, check: boolean): void => {
+    setTaskList([...taskList, { value: task, id: newKey, check: false }]);
```

Comment:
aqui vc pegou o valor check mas n passou para a funçao , tmb pq nesse caso o valor padrao é false , entao pode tirar o argumento check

```suggestion
  const addTask = (task: string, newKey: string): void => {
    setTaskList([...taskList, { value: task, id: newKey, check: false }]);
```